src/OpenApi/test/OpenApiGeneratorTests.cs

```diff
@@ -336,7 +337,7 @@ static void AssertBodyParameter(OpenApiOperation operation, string expectedName,
     public void AddsMultipleParameters()
     {
         var operation = GetOpenApiOperation(([FromRoute] int foo, int bar, InferredJsonClass fromBody) => { });
-        Assert.Equal(3, operation.Parameters.Count);
+        Assert.Equal(2, operation.Parameters.Count);
```

Comment:
Great! This test change does a great job reflecting the original bug that was fixed here.